test/vanilla-wildfly/test-app/extra-content/myscript.cli

```diff
@@ -0,0 +1,6 @@
+echo "Hi from CLI"
+:whoami
+if outcome==success of /system-property=foo:read-resource
```

Comment:
that's not related to your PR but sometimes I would like to tell to the CLI "ignore if the operation fails" in a simpler fashion. Something like

```
/system-property=foo:remove?
/system-property=foo:add(value=bar)
```

the `?` would ensure that the CLI operation ignores the failure if the operation fails and proceeds with the rest of the execution.